contrib-zypper/README.md

```diff
@@ -0,0 +1,35 @@
+# Zypper
+
+Defines [zypper][1] aliases.
+
+## Aliases
+
+* `zyp` # Zypper.
+* `zypc` # Cleans the cache.
+* `zypi` # Installs package(s).
+* `zypl` # Clean packages cache.
+* `zypq` # Displays package information.
+* 'zypre' # Refreshes repositories.
+* `zypx` # Removes package(s).
+* `zyps` # Searches for a package.
+* `zypv` # Verifies dependencies.
+* `zypu` # Updates packages.
+* `zypU` # Dist Upgrade.
+* `zypUd` # Dist Upgrade download only.
+* `zypUn` # Dist Upgrade without refreshing repositories.
+* `zyplp` # Lists patches.
+* `zypap` # Applies patch.
+* `zypaP` # Applies patches.
+* `zypar` # Adds repository.
```

Comment:
Would this make more sense as `zypRa` and then remove would be `zypRx` or `zypRr`?